src/gemdat/path.py

```diff
@@ -205,10 +206,130 @@ def free_energy_graph(F: np.ndarray,
                                'dijkstra-exp']
 
 
+def calculate_path_difference(path1: list, path2: list) -> float:
+    """Calculate the difference between two paths. This difference is defined
+    as the percentage of sites that are not shared between the two paths.
+
+    Parameters
+    ----------
+    path1 : list
+        List of sites defining the first path
+    path2 : list
+        List of sites defining the second path
+
+    Returns
+    -------
+    difference : float
+        Difference between the two paths
+    """
+
+    # Find the shortest and longest paths
+    if len(path1) <= len(path2):
+        shortest, longest = path1, path2
+    else:
+        shortest, longest = path2, path1
```

Comment:
Cool idea with the min/max! I think this should work:

```suggestion
    shortest, longest = sorted((path1,path2), key=len)
```

Reply:
This looks neat!